web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -10,6 +10,24 @@
       />
     </v-toolbar>
     <v-container
+      v-if="dandisetDoesNotExist"
```

Comment:
I don't think there's actually a need to use a new ref. If you're going for clarity, I'd think a comment would suffice, but I won't block on it.

```suggestion
      v-if="!(loading || currentDandiset)"
```

Reply:
I think I agree with @AlmightyYakob, fwiw. Though (and now we're at real risk of bikeshedding) I would probably express it as `!loading && !currentDandiset` only because that reads a little bit clearer to me ("ok so I'm done loading, but there's still no dandiset...").

Reply:
Agreed. For clarity, I went with a `computed` property instead of inlining `!loading && !currentDandiset` into the `v-if`, since you can't leave comments inside HTML tags.